src/pill/pill.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { PillProps } from "./types";
+import { Label, Wrapper } from "./pill.style";
+
+export const Pill = ({
+    type,
+    colorType = "black",
+    className,
```

Comment:
Since you are not doing anything to `className`, can let it fall under `otherProps`